tests/test_modeling_lxmert.py

```diff
@@ -246,11 +251,11 @@ def create_and_check_lxmert_model(
         result = model(input_ids, (visual_feats, bounding_boxes), return_dict=False)
         result = model(input_ids, (visual_feats, bounding_boxes), return_dict=True)
 
-        self.parent.assertEqual(result.last_hidden_state_l.shape, (self.batch_size, self.seq_length, self.hidden_size))
+        self.parent.assertEqual(result.language_output.shape, (self.batch_size, self.seq_length, self.hidden_size))
         self.parent.assertEqual(
-            result.last_hidden_state_v.shape, (self.batch_size, self.num_visual_features, self.hidden_size)
+            result.vision_output.shape, (self.batch_size, self.num_visual_features, self.hidden_size)
         )
-        self.parent.assertEqual(result.pooled_output_x_encoder.shape, (self.batch_size, self.hidden_size))
+        self.parent.assertEqual(result.pooled_output.shape, (self.batch_size, self.hidden_size))
```

Comment:
I hope you'll agree with me that this output is cleaner and simpler to understand for users :)